.github/workflows/test.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           cache: npm
-          node-version: 20
+          node-version: lts/*
```

Comment:
Do we want to rely on `lts/*` and not control when we are ready to upgrade node?

Reply:
Could we potentially get different `tsc` results depending on the node version we use for `npm run validate:ts`. Do we want to run this with a matrix of node versions?

Approving in the meantime as this should be a different issue (in case we want).

Reply:
Typescript is supposed to give the same output no matter the NodeJS version